app/views/users/show.html.erb

```diff
@@ -1,2 +1,30 @@
-<h1>Here will display user<h1>
+<section>
+  <div class="user-card">
+    <div class="photo">
+      <img class="user-photo" src=<%=@user.photo%>>
+    </div>
+    <div class="user-info">
+      <%= link_to user_path(@user.id) do%>
+        <h2 class="name"><%= @user.name %></h2>
+      <% end %>
+      <p class="post-numbers"> Number of post: <%= @user.posts_counter %> </p> 
+    </div>
+  </div>
 
+  <div class="bio">
+    <h4> Bio: </h4>
+    <p><%= @user.bio %>
+  </div>
+  
+  <div class="user-last-posts">
+    <%@user.recent_posts.each_with_index do |post, idx|%>
+      <div class="user-post">
+        <h4>Post # <%= idx + 1 %></h4>
+        <p><%=post.text%></p>
+      </div>
+    <%end%>
+  </div>
+  <div class="button-container">
+  <button>See all post</button>
```

Comment:
- [ ] I should be able to click this button and links opens the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png)